eisbuk-admin/src/components/atoms/CustomerList/CustomerDialog.tsx

```diff
@@ -0,0 +1,284 @@
+  __sendBookingsEmailId__,
+} from "./__testData__/testIds";
+import { __customersDialogId__ } from "@/__testData__/testIds";
+import Button from "@material-ui/core/Button";
+
+interface Props {
+  customer: Customer | null;
+  onClick?: (customer: Customer) => void;
+  onClose: () => void;
+}
+
+const CustomerDialog: React.FC<Props> = ({ onClose, customer }) => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  return (
+    <Dialog
+      data-testid={__customersDialogId__}
+      open={Boolean(customer)}
+      onClose={onClose}
+    >
+      <Card>
+        <CardContent className={classes.CardContent}>
+          <Box className={classes.Box}>
+            <EisbukAvatar
+              {...{ ...customer!, className: classes.BiggerAvatar }}
+            />
+            <div className={classes.NameContainer}>
+              <Typography variant="h4" className={classes.Name}>
+                {customer?.name}
+              </Typography>
+              <Typography variant="h4" className={classes.Name}>
+                {customer?.surname}
+              </Typography>
+            </div>
+          </Box>
+          <br />
+          <Typography variant="h6" component="div">
+            <span className={classes.BoldLabel}>
+              {t(CustomerLabel.Category)}:{" "}
+            </span>
+            {capitalizeFirst(
+              t(CategoryLabel[customer?.category || ""])
+            ).replace("-", " ")}
+          </Typography>
+          <Typography variant="h6">
+            <span className={classes.BoldLabel}>
+              {t(CustomerLabel.Email)}:{" "}
+            </span>
+            {customer?.email}
+          </Typography>
+          <Typography variant="h6">
+            <span className={classes.BoldLabel}>
+              {" "}
+              {t(CustomerLabel.DateOfBirth)}:{" "}
+            </span>
+            {customer?.birthday}
+          </Typography>
+          <Typography variant="h6">
+            <span className={classes.BoldLabel}>
+              {" "}
+              {t(CustomerLabel.Phone)}:{" "}
+            </span>
+            {customer?.phone}
+          </Typography>
+          <Typography variant="h6">
+            <span className={classes.BoldLabel}>
+              {" "}
+              {t(CustomerLabel.CardNumber)}:{" "}
+            </span>
+            {customer?.subscriptionNumber || "-"}
+          </Typography>
+          <Typography variant="h6">
+            <span className={classes.BoldLabel}>
+              {" "}
+              {t(CustomerLabel.MedicalCertificate)}:{" "}
+            </span>
+            {customer?.certificateExpiration}
+          </Typography>
+          <Typography variant="h6">
+            <span className={classes.BoldLabel}>
+              {" "}
+              {t(CustomerLabel.CovidCertificateReleaseDate)}:{" "}
+            </span>
+            {customer?.covidCertificateReleaseDate}
+          </Typography>
+        </CardContent>
```

Comment:
This should be automated (with mapping function). Additional changes to i18n logic might be needed for this, but it's worth it.

Reply:
There should be a way to assert the customer is truthy. I would do a `customer: customerFromProps` (in props, used only for boolean condition` and `const customer = customerFromProps!` to assert truthy only once and remove the aannoying `!` or `?` down the line.